index.js

```diff
@@ -99,24 +93,22 @@ controller.hears(['hello'], ['message_received', 'direct_message', 'direct_menti
 })
 
 function getUsers (callback) {
-  web.users.list(token, (err, response) => {
-    if (!err) {
-      response.members.forEach((user) => {
-      // need to get id for each user and pass it into the save below
-        controller.storage.users.save(user, function (err) {
-          if (err) console.log(err + ' a')
-        })
+  web.users.list((err, response) => {
+    if (err) callback(err)
```

Comment:
this should be `if (err) return callback(err)` otherwise the code underneath will continue to execute and would probably cause the program to crash